docs/tools/vega-wallet/cli-wallet/latest/create-wallet.md

```diff
@@ -7,23 +7,21 @@ hide_title: false
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-To download Vega Wallet and create your wallet, follow the step-by-step instructions below. These instructions cover version 0.12, the latest release, and also work for version 0.11. You can refer to [documentation for 0.10.0 or earlier](/docs/tools/vega-wallet/cli-wallet/old-versions/v0-10-0/create-wallet-v0-10) if you need it. 
-
-This software a work-in-progress and is frequently updated, and does not yet have a user interface. 
+To download Vega Wallet and create your wallet, follow the step-by-step instructions below. This software a work-in-progress and is frequently updated. These instructions cover version 0.14, the latest release.
 
 Note: If you are looking for instructions for connecting your hardware wallet to MetaMask, see [MetaMask's guide](https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet).
 
 Use the following instructions in command line. Below, you'll see commands in the code blocks for each operating system. Copy those instructions and paste them into your command line interface.
 
 :::caution
-You must to use Vega Wallet version 0.9.2 or newer to connect to Vega Mainnet. We recommend always using the latest released version of Vega Wallet. 
+You must use Vega Wallet version 0.9.2 or newer to connect to the Vega mainnet. We recommend always using the latest released version of Vega Wallet. 
```

Comment:
```suggestion
You must use Vega Wallet version 0.9.2 or newer to connect to the Vega mainnet. We highly recommend always using the latest released version of Vega Wallet. 
```

Also, the thing is that some protobuf definition have been updated since then, so some of the transaction won't pass the sanity checks... I wonder if we shouldn't say something like...

> Previous version of the wallet may rely on outdated transaction format that may not be compatible anymore with the deployed network. We highly recommend you to use the latest version. Otherwise, you might encounter errors when sending transactions.


What do you think ?

Reply:
Because mainnet is being maintained with version 0.47, I'm going to park this suggestion for now but I think it will  be true in the future.